### DISCUSSION TITLE
Is there any way we can generate images of good pore connectivity at low porosity value like 20%?

### DISCUSSION THREAD

**Discussion by himanshu-sahu674** [PMEAL/porespy#752] [Q&A] [answered]
Hello Sir,
I tried to predict the permeability of generated images from porespy's blob function with porosity value 20% and 60% with varying blobiness from 1 to 3.
 When I'm using 60% porosity I'm getting good connectivity of pores and permeability value while using 20% there is not good connectivity in generated image and also getting zero permeability.
But in oil and gas field it is frequently observed that at low porosity like 20% there is good connectivity thus good permeability.

So is there any way we can generate images of good pore connectivity at low porosity value like 20%?

PFA image generated using porespy at 20% porosity and 60% porosity

![generated1](https://user-images.githubusercontent.com/85443615/201533476-dc4689c4-168a-432f-b574-e2d47eddcdd7.png)
![generated2](https://user-images.githubusercontent.com/85443615/201533497-78607639-03c9-4399-b308-3103c0f6b4ef.png)

Thank You

**Comment by jgostick** [comment, marked as answer]
To have percolation of the void space at the low of a porosity, there must be some sort of correlation, like anisotropy.  You can try setting blobiness to [3, 1], which will create elongated blobs.  You can also play with the fractal noise based functions, which create hierarchy that might allow connectivity to persist.  Although these functions don't allow for anisotropic images

Example [here](https://porespy.org/examples/generators/reference/fractal_noise.html)
